### PR TITLE
Bug fixes for CLIQZ 1.2.0

### PR DESCRIPTION
**Pull request by timoteipalade**

### CODE REVIEW

File: Client/Info.plist [hunk 1]
```diff
@@ -38,7 +38,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>2</string>
```
Comment (mahmoud-adam85) [inline Client/Info.plist:41]: build number should be changed across all target not only for client

File: Client/Cliqz/Frontend/Browser/BrowserViewControllerExtension.swift [hunk 2]
```diff
@@ -291,6 +291,9 @@ extension BrowserViewController: ControlCenterViewDelegate {
     }
 
 	func showHint(type: HintType) {
+        if type == HintType.Antitracking && self.urlBar.isAntiTrackingButtonHidden(){
```
Comment (mahmoud-adam85) [inline Client/Cliqz/Frontend/Browser/BrowserViewControllerExtension.swift:294]: `showHint` should be called only if the count is > 0
I don't know how this happen on the fresh tab
@naira-cliqz

Comment (timoteipalade) [reply Client/Cliqz/Frontend/Browser/BrowserViewControllerExtension.swift:294]: I don't undestand.

File: Client/Info.plist (hunk 1 above)
Comment (timoteipalade) [reply Client/Info.plist:41]: Ok, I will change them all

**Pull request merged**